### PR TITLE
Updated docker-compose.yml

### PR DESCRIPTION
**Pull request by clintonb**
- Updated to schema version 2
- Removed unused rabbitmq service
- Specifying credentials for MySQL user

### CODE REVIEW

**Comment by clintonb** [comment]
FYI @cpennington @bderusha @peter-fogg

File: Makefile [hunk 1]
```diff
@@ -80,10 +80,10 @@ push_translations:
 	tx push -s
 
 start-devstack:
-	docker-compose --x-networking up
+	docker-compose up
 
 open-devstack:
-	docker-compose --x-networking up -d
+	docker-compose up -d
```
Comment (peter-fogg) [inline Makefile:86]: Why are we removing the experimental networking? Or maybe I should be asking why we were using it in the first place?

**Comment by peter-fogg** [comment]
When do you sleep? :+1:

File: Makefile (hunk 1 above)
Comment (clintonb) [inline Makefile:86]: This functionality is now built in. By using version 2 of the docker compose file format, we get networking "for free".

Comment (clintonb) [inline Makefile:86]: See https://github.com/docker/compose/releases/tag/1.6.0-rc1. There is a mention there about "networking".

Comment (peter-fogg) [inline Makefile:86]: Neat.

**Comment by cpennington** [comment]
Does travis have the right docker version?

**Comment by cpennington** [comment]
With the new format, does it make sense to specify a volume so that the databases can be persistent?

**Comment by bderusha** [comment]
LGTM :+1:

**Comment by clintonb** [comment]
@cpennington the new format does not overcome the issues with Mac OS X permissions. Just specifying a volume, even to a directory in my user's home directory doesn't work. I'm going to merge these basic changes to get things moving again. Hopefully Docker for Mac (still in beta) will improve the volume support.

**Pull request merged**